ee.ria.qdigidoc4.yml

```diff
@@ -35,27 +35,25 @@ modules:
     buildsystem: cmake-ninja
     sources:
       - type: archive
-        url: https://github.com/google/flatbuffers/archive/refs/tags/v25.2.10.zip
-        sha256: 75ffbce7d32f8218b5faec86ae2f6397c7ca810605dc710dfa9c146b9df9e3e9
+        url: https://github.com/google/flatbuffers/archive/refs/tags/v25.9.23.tar.gz
+        sha256: 9102253214dea6ae10c2ac966ea1ed2155d22202390b532d1dea64935c518ada
 
   - name: xmlsec
     buildsystem: autotools
     sources:
       - type: archive
-        url: https://github.com/lsh123/xmlsec/releases/download/1.3.7/xmlsec1-1.3.7.tar.gz
-        sha256: d82e93b69b8aa205a616b62917a269322bf63a3eaafb3775014e61752b2013ea
+        url: https://github.com/lsh123/xmlsec/archive/refs/tags/1.3.9.tar.gz
+        sha256: 534f50743b2fa3ebf3c05917a293c73f255c823dd0285f02622260084317c605
```

Comment:
This accidental change doesn't seem to be causing any issues. According to the readme you need to run `./autogen.sh` if you build "from GitHub" but I guess `buildsystem: autotools` takes care of that?